test/integration/patients/patient/patient-flow.js

```diff
@@ -1035,39 +1035,42 @@ context('patient flow page', function() {
     cy
       .routesForPatientAction()
       .routeFlow(fx => {
-        const flowActions = _.sample(fx.data.relationships.actions.data, 3);
-
-        fx.data.id = '1';
-
-        _.each(flowActions, (action, index) => {
-          action.id = `${ index + 1 }`;
+        fx.data = mergeJsonApi(testFlow, {
+          relationships: {
+            state: getRelationship(stateInProgress),
+          },
+          meta: {
+            progress: {
+              complete: 0,
+              total: 3,
+            },
+          },
         });
 
-        fx.data.relationships.actions.data = flowActions;
-        fx.data.relationships.state.data.id = '33333';
-
-        fx.data.meta.progress.complete = 0;
-        fx.data.meta.progress.total = 3;
-
         return fx;
       })
       .routePatientByFlow()
       .routeFlowActions(fx => {
-        fx.data = _.first(fx.data, 3);
-
-        _.each(fx.data, (action, index) => {
-          action.id = `${ index + 1 }`;
-        });
-
-        fx.data[0].relationships.state.data.id = '22222';
-        fx.data[1].relationships.state.data.id = '22222';
-        fx.data[2].relationships.state.data.id = '22222';
-
-        fx.data[0].relationships.flow.data = { id: '1' };
-        fx.data[1].relationships.flow.data = { id: '1' };
-        fx.data[2].relationships.flow.data = { id: '1' };
-
-        fx.included = _.reject(fx.included, { type: 'flows' });
+        fx.data = [
```

Comment:
I actually think this might be a good case for `getActions` since all the relationships are the same.

Reply:
Updated in https://github.com/RoundingWell/care-ops-frontend/pull/1249/commits/38ecf9ddfd0517a6583e7ff1cf53b77c0bc39fff.